numpy/lib/type_check.py

```diff
@@ -252,6 +252,9 @@ def isreal(x):
     If element has complex type with zero complex part, the return value
     for that element is True.
 
+    .. note:: Returns all True for input elements which are not of complex
+    type.
```

Comment:
This syntax won't work, can you change it to look like the `.. note::` directive in https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#paragraph-level-markup?

Reply:
ah!! thanks for pointing, fixed!